src/maker/api.rs

```diff
@@ -277,7 +278,7 @@ impl Maker {
         for funding_info in &message.confirmed_funding_txes {
             // check that the new locktime is sufficently short enough compared to the
             // locktime in the provided funding tx
-            let locktime = read_contract_locktime(&funding_info.contract_redeemscript)?;
+            let locktime:locktime::relative::LockTime = read_contract_locktime(&funding_info.contract_redeemscript)?;
```

Comment:
This is not a way to get a value of  particular type in rust.
For `locktime` to be of desired type  -> `read_contract_locktime` must return value of type `locktime::relative::LockTime`.

Correct Way: 
``` rust
pub fn read_contract_locktime(redeemscript: &Script) -> Result<u16, ProtocolError> 
  |
  change this to 
  |
  pub fn read_contract_locktime(redeemscript: &Script) -> Result<Locktime, ProtocolError> 
  ```
  
 then-> 
 ``` rust
  let locktime = read_contract_locktime(&funding_info.contract_redeemscript)?;
`locktime`  will be of type `Locktime`.
  ```